src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -779,6 +778,18 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji addEmoji(String name, BufferedImage image){
```

Comment:
The built-in `Image` class would be better since @MinnDevelopment confirmed that it follows the same formatting as avatars